packages/separation

```diff
@@ -0,0 +1,5 @@
+type        = theme
+repository  = https://github.com/roflcopter4/separation
+maintainer  = RoflCopter4 <brendan.leason.4@gmail.com>
+description = An oh-my-fish theme for those who like the output of each 
+              command to be clearly distinguishable from the rest. 
```

Comment:
I don't think multi-line descriptions will work in the manifest parser. You might want to put the description all on one line here.